src/lib/autoping.ts

```diff
@@ -0,0 +1,41 @@
+import { ForumChannel, GuildChannel, TextChannel, ThreadChannel } from "discord.js";
+
+import { CustomClient } from "./client";
+
+export async function threadCreated(client: CustomClient, thread: GuildChannel) {
+  if (thread.parentId === null) {
+    return;
+  }
+  if (!(thread instanceof ThreadChannel)) {
+    return;
+  }
+  const forum = await thread.guild.channels.fetch(thread.parentId);
+  if (!(forum instanceof ForumChannel)) {
+    return;
+  }
+  const appliedTags = forum.availableTags
+    .filter((tag) => thread.appliedTags.includes(tag.id))
+    .map((tag) => tag.name.toLowerCase().match("[a-z0-9_]*")?.join("").trim());
```

Comment:
This `.match` regex pattern is only described and used here and nowhere else, and there doesn't appear to be a check when inputting to this command anywhere?
This will also only match the first (if any) matches of the filter, don't know if this is intended or not.